SparkPost/2web2ui
# EO-754 Add recommended actions messages for Signals Engagement Rate by cohort

### What Changed
 - Added recommended actions messages for engagement rate by cohort. Values from here: https://docs.google.com/spreadsheets/d/1AOcX1eQ6flweFV0O3vFTaYyG0h_QC5e7LHLvS7Mjdsg/edit#gid=92043718.
The total engagement rate error message is unique because it requires a facet specific link. 
- Added component that generates the recommended actions messages. 
- Added tests.

### How To Test
 - [Point your local upstreams to staging or production](https://confluence.int.messagesystems.com/display/ENG/Configuring+Openresty+Upstreams)
- Log into appteam or account 108 for data ( production account 108 works best because it has actual data)
- Visit v2 Engagement Recency dashboard
- Click on one of the accounts
- Click on `Engagement Rate` tab.
- Check that all recommendations are the correct ones. 
### To Do
